### PR TITLE
docs: add commit_files table to the schema documentation

### LINKED ISSUE #351: Documentation: add commit_files table

### PR DESCRIPTION
**Pull request by mcarmonaa**
Closes #351 

Signed-off-by: Manuel Carmona <manu.carmona90@gmail.com>

### CODE REVIEW

File: docs/using-gitbase/schema.md [hunk 1]
```diff
@@ -250,4 +265,4 @@ blob_hash string FK >- blobs.blob_hash
 
  -->
 
-![gitbase schema](/docs/assets/gitbase-db-diagram.png)
\ No newline at end of file
+![gitbase schema](/docs/assets/gitbase-db-diagram.png)
```
Comment (ajnavarro) [inline docs/using-gitbase/schema.md]: you need to change the diagram too.

**Comment by mcarmonaa** [comment]
@ajnavarro  the new diagram is already added

**Comment by erizocosmico** [comment]
@ajnavarro what tool did you use to generate the ER diagram? There was a comment with the code to generate it, right? If we add there also the tool I think it's easier than having to use MySQL workbench and it's better for version control.

**Comment by ajnavarro** [comment]
@erizocosmico the tool that I use has a 10 tables limit. Now we have 11.

**Comment by erizocosmico** [comment]
Nvm then

**Comment by ajnavarro** [comment]
@mcarmonaa the last thing: https://stackoverflow.com/questions/14946161/how-to-draw-relationship-lines-between-columns

To be able to see relationships between columns.

**Comment by mcarmonaa** [comment]
@ajnavarro rebased

**Pull request merged**